test/trails_test3_handler.erl

```diff
@@ -0,0 +1,15 @@
+-module(trails_test3_handler).
```

Comment:
For this module to be tested, you need to add it to `trails_SUITE:basic_trails_routes/1`… or create a new test function for it.